HDF5Examples/FORTRAN/H5T/test-pc.sh

```diff
@@ -242,12 +255,12 @@ do
     return_val=`expr $status + $return_val`
 done
 
-$H5FC $srcdir/h5ex_t_convert.F90 -o h5ex_t_convert
+#compileout $top_srcdir/$currentpath/h5ex_t_convert.F90 -o h5ex_t_convert
```

Comment:
Commented out now?

Reply:
Fortran is the same as C except that file doesn't exist in Fortran - left the comment to determine if it is needed.